.github/workflows/cleardown-terraform-state.yaml

```diff
@@ -0,0 +1,41 @@
+name: Cleardown Terraform state
```

Comment:
I am not sure we need this yaml? I think it is superceded by the cleardown-tf-state/action above.